components/renku_data_services/project/api.spec.yaml

```diff
@@ -771,6 +779,15 @@ components:
     IsTemplate:
       description: Shows if a project is a template or not
       type: boolean
+    SecretsMountDirectory:
+      description: The location where the secrets will be provided inside sessions, if left unset it will default to `/secrets`.
+      type: string
+      minLength: 1
+      default: "/secrets"
+      example: "/secrets"
```

Comment:
1. Weird characters do not seem to break it. For example, using `~` work, it is just not interpreted as `$HOME`. Validation could be hard to do, since failure happens when there is a clash with other mount locations which will not be known at validation time.
2. Using a leading `/` works and creates an absolute mount. If you do `a / b` with `PurePosixPaths` and `b` is absolute then `a / b = b` which is what the user requested.

Reply:
API Spec updated in 9065bbf46e3ee0858a422aada9c752e98b5d5ff6.